tests/vmi_multus_test.go

```diff
@@ -1230,3 +1256,33 @@ func checkSriovEnabled(virtClient kubecli.KubevirtClient, sriovResourceName stri
 	}
 	return false
 }
+
+func waitForVMIGuestAgentReport(virtClient kubecli.KubevirtClient, vmiNamespace, vmiName string, numOfinterfaces int) error {
```

Comment:
This is a controversial helper and although it may resolve some flakiness in this test, it does fit under a more focus discussion.

I suggest you use an `eventually` on the network name assertion as an intermediate solution.

Reply:
please see https://github.com/kubevirt/kubevirt/pull/4491
will use the same on the 3 usages here